scrapy/commands/crawl.py

```diff
@@ -19,10 +17,8 @@ def add_options(self, parser):
         ScrapyCommand.add_options(self, parser)
         parser.add_option("-a", dest="spargs", action="append", default=[], metavar="NAME=VALUE",
                           help="set spider argument (may be repeated)")
-        parser.add_option("-o", "--output", metavar="FILE",
+        parser.add_option("-o", "--output", metavar="FILE", action="append",
```

Comment:
We should extend the documentation of this option to cover the usage of `:` to specify the output format.

It should be specially clear that, in order to use an output file name that contains one or more colons, specifying the output format is mandatory. For example, to write to `data:countries.csv` in CSV format, you must write `data:countries.csv:csv`.

We could support `data:countries.csv:` instead (if the format is an empty string, the file extension is used), but I’m not sure saving a few characters is worth making the implementation more complex.

Reply:
NIce case, I hadn't thought of that, thanks!

Reply:
I was thinking about _where_ to put this note, since the documentation about the commands does not go that deep as to mention `-o/-t`, but then I realized that the warning in this case is pretty self-explanatory IMHO:
```
$ scrapy runspider example.py -o data:countries.csv
Usage
=====
  scrapy runspider [options] <spider_file>

runspider: error: Unrecognized output format 'countries.csv', set one after a colon using the -s option (i.e. -o <URI>:<FORMAT>) or as a file extension, from the supported list ('json', 'jsonlines', 'jl', 'csv', 'xml', 'marshal', 'pickle')
```

What do you think @Gallaecio ?

Reply:
Yeah, I guess you are right.